GliaWidgets/Component/Connect/ConnectView.swift

```diff
@@ -103,45 +117,30 @@ class ConnectView: UIView {
 
     private func show(animated: Bool) {
         guard !isShowing else { return }
-        UIView.animate(withDuration: animated ? 0.5 : 0.0,
-                       delay: 0.0,
-                       usingSpringWithDamping: 0.8,
-                       initialSpringVelocity: 0.7,
-                       options: .curveEaseInOut,
-                       animations: {
-                        self.transform = .identity
-                       }, completion: nil)
+        UIView.animate(
```

Comment:
If animated is false, shouldn't the transform happen on its own, not inside `UIView.animate`?

Reply:
to manage this we previously used zero duration, so I decided to let it as is.
```swift
withDuration: animated ? 0.5 : 0.0,
```
do you think this is good time to rework it and use?